package.json

```diff
@@ -72,7 +75,14 @@
     "build": "yarn && yarn run icons && gatsby build",
     "build-serve": "yarn build && gatsby serve",
     "develop": "yarn && yarn run icons && gatsby develop",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fix-code": "prettier-eslint --write 'src/**/*.{js,jsx}' ",
+    "fix-styles": "prettier-stylelint --write 'src/**/*.{css,scss}' ",
+    "lint-code": "eslint 'src/**/*{js,jx}'",
```

Comment:
I noticed after adding in the Atom Prettier plugin that there were some changes in files like `stylelint.config.js`, `gatsby-config.js` and `utils/build-icons.js`. Maybe we could make `lint-code` and `fix-code` a bit more generic to catch any/all JS/JSX files throughout the entire directory; instead of just `/src`? Also, I'd recommend something like `lint-js` since it's specific to JS files and "code" is a bit of an all encompassing word; since we use `fix-styles` for styles.